flink-libraries/flink-table/src/main/scala/org/apache/flink/table/api/table.scala

```diff
@@ -66,6 +66,15 @@ class Table(
   private val tableSchema: TableSchema = new TableSchema(
     logicalPlan.output.map(_.name).toArray,
     logicalPlan.output.map(_.resultType).toArray)
+  
+  // map to save the all windows information registered in the table
+  private[flink] var windowPool: Map[Expression, GroupWindow] = Map()
+
+  // get the windowPool from the existing table
```

Comment:
better using /** xxxx */ for single line comment of a function

see http://docs.scala-lang.org/style/scaladoc.html